frontend/src/translations.json

```diff
@@ -1,6 +1,77 @@
 {
   "pl": {
-
+    "common": {
+      "about": "O nas",
+      "home": "Strona glowna",
+      "login": "Login",
+      "new_route": "Dodaj tras",
+      "routes": "Trasy",
+      "intro": "Polska Akcja Humanitarna jest polską organizacją pozarządową działającą w Polsce i innych krajach. Jego misją jest \"jest uczynienie świata lepszym miejscem poprzez łagodzenie ludzkiego cierpienia i promowanie wartości humanitarnych\"."
+    },
+    "routes": {
+      "please_correct_errors": "Please correct the following error(s):",
+      "date": "Date:",
+      "description": "Description:",
+      "from": "From:",
+      "destination": "Desitination:",
+      "starting_mileage": "Starting Mileage:",
+      "ending_mileage": "Ending Mileage:",
+      "submit": "Submit",
+      "validation_error": "{field} is required",
+      "no_driver_routes": "You have no routes stored in database. You can add a new route by choosing option from the menu.",
+      "from_to": "From {from} to {destination}"
+    },
+    "login": {
+      "user": "User: %{username}",
+      "username": "Username",
+      "password": "Password",
+      "username_required": "Username is required",
+      "password_required": "Password is required"
+    },
+    "refresh": {
+      "new_version": "A new version of website is available.",
+      "reload": "Reload"
+    },
+    "header": {
+      "polish_humanitarian_action": "Polska Akcja Humanitarna"
+    }
+  },
+  "ua": {
+    "common": {
+      "about": "Про",
+      "home": "Домашня сторінка",
+      "login": "Логін",
+      "new_route": "Додати новий маршрут",
+      "routes": "Маршрути",
+      "intro": "Польська гуманітарна акція - це польська неурядова організація, яка працює у Польщі та інших країнах. Його завдання \"зробити світ кращим місцем шляхом полегшення людських страждань та просування гуманітарних цінностей\"."
+    },
+    "routes": {
+      "please_correct_errors": "Please correct the following error(s):",
```

Comment:
English is the default language. Other languages should only have the translated fields specified here, there is no need to repeat the English messages.

Reply:
I think that there is an error, where if keys are missing from translation, entire translation engine stops translating. It could be matter of configuration... not sure..

Reply:
Thanks, but anyway I gonna correct this in the next task